extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergWriteInstructions.java

```diff
@@ -42,11 +45,28 @@ public boolean createTableIfNotExist() {
     public abstract Optional<Boolean> verifySchema();
     // @formatter:on
 
+    /**
+     * A one-to-one {@link Map map} from Deephaven to Iceberg column names to use when writing deephaven tables to
+     * Iceberg tables.
+     */
+    public abstract Map<String, String> dhToIcebergColumnRenames();
+
+    /**
+     * The inverse map of {@link #dhToIcebergColumnRenames()}.
+     */
+    @Value.Lazy
+    public Map<String, String> icebergToDhColumnRenames() {
```

Comment:
I see you do have the inverse, but I think this should be the _source_ of the data, and not a view of it. I think we should also consider if we even want to provide as a public helper, or if it should be only for internal use.

Reply:
Same response as https://github.com/deephaven/deephaven-core/pull/5989#discussion_r1792187149